src/dryad.lisp

```diff
@@ -150,26 +171,30 @@ NOTE: In the basic implementation, these messages must be waiting for the DRYAD
                    (right-address (blossom-edge-target-vertex (first reply)))
                    (left-member (member left-address emitted-addresses :test #'address=))
                    (right-member (member right-address emitted-addresses :test #'address=))
-                   (ids (list (gethash left-address  (dryad-ids dryad))
-                              (gethash right-address (dryad-ids dryad)))))
+                   (address-pair (list left-address right-address)))
               (cond
                 ((and left-member right-member)
                  nil)
                 ((and (not left-member) (not right-member))
                  (push left-address emitted-addresses)
                  (push right-address emitted-addresses)
-                 (push ids pairs))
+                 (push address-pair pairs))
                 (t
                  (error "Two distinct match edges laid claim to the same vertex.")))))
           (process-continuation dryad
                                 `(PROCESS-PAIRS ,pairs)
                                 `(WIND-DOWN)))))))
 
 (define-process-upkeep ((dryad dryad) now) (PROCESS-PAIRS pairs)
-  "Iterates through `PAIRS' and sends corresponding REAP messages."
-  (dolist (pair pairs)
-    (send-message (dryad-match-address dryad)
-                  (make-message-reap :ids pair))))
+  "Iterates through `PAIRS' of addresses and sends corresponding WILT and REAP messages."
+  (dolist (address-pair pairs)
+    (log-entry :entry-type 'processing-pair
+               :pair address-pair)
+    (send-message-batch #'make-message-wilt address-pair)
```

Comment:
This surprises me. Before introducing this line, who was wilting? Are they not wilting anymore? Is this some extra safety? Can the other wilt point, wherever it is, be removed?

Reply:
no wilting was happening in anatevka, which is not critical bc it’s offline